pegasus/cache/i18n/en-US.yml

```diff
@@ -1532,7 +1532,7 @@
   header_home: "Home"
   header_my_dashboard: "My Dashboard"
   header_courses: "Courses"
-  header_course_catalog: "Course Catalog"
+  header_course_catalog: "Curriculum Catalog"
```

Comment:
Have you updated this in the [i18n gsheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0)? You can update it here but it'll get overriden with the next scoop if you haven't added it to that gsheet. More info at [this README](https://github.com/code-dot-org/code-dot-org/blob/staging/i18n/README.md) if you're interested!

Reply:
Just updated it in the gsheet!

Reply:
Thank you, Dani, and thanks for the README link, Bethany!